app/features-json/artifact_json_controller.rb

```diff
@@ -0,0 +1,86 @@
+require_relative "./view_models/artifact_view_model"
+
+module FastlaneCI
+  # Controller for providing all data relating to artifacts
+  class ArtifactJSONController < APIController
```

Comment:
Cool, auth in enabled on these end points ❤️

Reply:
What do you mean?

Reply:
I mean, these end points require JWT auth in order to allow you to access them, which I think is the right move here.